deb/openmediavault/usr/share/openmediavault/notification/sink.d/README

```diff
@@ -13,9 +13,12 @@ OMV_NOTIFICATION_SUBJECT
 OMV_NOTIFICATION_DATE
 OMV_NOTIFICATION_MESSAGE_FILE
 
-Additionaly the scripts/executables are executed with the following command
+Additional the scripts/executables are executed with the following command
```

Comment:
I think should be changed to `Additionally`

Reply:
Thx, didn't check the result after the IDE has changed that typo automatically. Now changed to `Additionally` manually.